Libs/Gtk-3.0/Classes/ScrolledWindow.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using GObject.Native;
+
+namespace Gtk
+{
+    public partial class ScrolledWindow
+    {
+        public static ScrolledWindow New(Adjustment? hAdjustment = null, Adjustment? vAdjustment = null)
+        {
+            IntPtr ptr = Native.ScrolledWindow.Instance.Methods.New(
```

Comment:
The wrapper checks if an object is already known for this pointer and afterwards calls the constructor per reflection. But as we are creating a new instance we can omit this check as we now there is no other instance and direcly call a constructor.